pmm-app/src/pmm-add-instance-app-panel/AddInstance/AddRemoteInstance/AddRemoteInstance.tsx

```diff
@@ -213,32 +203,31 @@ const AddRemoteInstance = props => {
               <span className="description">Port your service is listening on</span>
             </div>
             <div className="add-instance-panel">
-              <InputField form={form} name="username" placeholder="Username" required={true} />
+              <InputField name="username" placeholder="Username" required={true} />
```

Comment:
`true` is the default value for props passed to components, so syntax should be just `required`. There is also a widely adopted [rule](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-boolean-value.md), which I encourage to use.

Reply:
Thanks, great comment, I'll add it to eslint